package.json

```diff
@@ -2,22 +2,31 @@
   "name": "music-box-interactive-client",
   "private": true,
   "version": "2.3.6",
+  "main": "dist/main/main.js",
   "type": "module",
   "scripts": {
     "dev": "vite",
     "debug-dev": "vite --debug hmr",
     "build": "vite build",
     "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\""
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "e:start": "electron-vite preview --outDir=dist",
+    "e:dev": "electron-vite dev --outDir=dist",
+    "e:package": "electron-vite build --outDir=dist && electron-forge package",
+    "e:make": "electron-vite build --outDir=dist && electron-forge make",
+    "e:build": "electron-vite build --outDir=dist"
   },
   "dependencies": {
     "axios": "^1.6.2",
     "better-react-mathjax": "^2.0.3",
     "bootstrap": "^5.3.2",
     "d3": "^7.8.5",
+    "electron-squirrel-startup": "^1.0.0",
+    "is-electron": "^2.2.2",
     "open-iconic": "^1.1.1",
     "prop-types": "^15.8.1",
+    "pyodide": "^0.25.0",
```

Comment:
Needed to check python version. is there a way to enforce a conditional dependency so its not necessary to build the web app?